extension/intents/tabs/tabs.toml

```diff
@@ -262,3 +262,27 @@ match = """
 
 [[tabs.moveSelectedToNewWindow.example]]
 phrase = "Move selected tabs to new window"
+
+[tabs.collectAllTabs]
+description = "Collect all tabs by url"
+match = """
+  collect (all | the|) tabs in all (the|) windows [currentWindow=false]
```

Comment:
I'd add `(in | from)` here, and use `window{s}` to make the pluralization optional.